### PR TITLE
add language around forcing disclosure of some SD-JWT properties 

### LINKED ISSUE #285: Add a requirement on SD-JWT that `credentialStatus` should always be visible (plain).
Hi.

There should be a requirement on `SD-JWT` that `credentialStatus` should always be plain and cannot be disclosable.
If this requirement is not there, there is the possibility for a revoked credential not be properly verified due to the presentation not disclosing the `credentialStatus`.

### PR DESCRIPTION
**Pull request by decentralgabe**
fix #285 
I will separately need to update the respec plugin to take into account these properties

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/pull/288.html" title="Last updated on Aug 6, 2024, 5:33 PM UTC (e71831a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/288/f083eaa...e71831a.html" title="Last updated on Aug 6, 2024, 5:33 PM UTC (e71831a)">Diff</a>

### CODE REVIEW

**Comment by decentralgabe** [comment]
cc: @goncalo-frade-iohk

File: index.html [hunk 1]
```diff
@@ -447,6 +447,13 @@ <h2 id="securing-with-sd-jwt">Securing JSON-LD Verifiable Credentials with SD-JW
                 A [=conforming SD-JWT verifier implementation=] MUST use [[SD-JWT]] to verify [=conforming JWS
                 documents=] that use this media type.
             </p>
+            <p>
+                When securing verifiable credentials with [[SD-JWT]] implementers MUST ensure that
+                properties necessary for the validation and verification of a credential are NOT selectively
+                disclosable. Such properties include but are not limited to <a data-cite="VC-DATA-MODEL-2.0#status">
+                <code>credentialStatus</code></a> and <a data-cite="VC-DATA-MODEL-2.0#data-schemas">
+                <code>credentialSchema</code></a>.
```
Comment (TallTed) [inline index.html:455]:
```suggestion
                When securing verifiable credentials with [[SD-JWT]] implementers MUST ensure that
                properties necessary for the validation and verification of a credential are NOT selectively
                disclosable (i.e., such properties MUST be disclosed). These properties include but are not
                limited to <a data-cite="VC-DATA-MODEL-2.0#status"><code>credentialStatus</code></a>
                and <a data-cite="VC-DATA-MODEL-2.0#data-schemas"><code>credentialSchema</code></a>.
```

File: index.html [hunk 2]
```diff
@@ -513,6 +520,13 @@ <h2 id="securing-vps-sd-jwt">Securing JSON-LD Verifiable Presentations with SD-J
                 Credentials in verifiable presentations MUST be secured.
                 These credentials are secured using SD-JWT in this case.
             <p>
+            <p>
+                When securing verifiable presentations with [[SD-JWT]] implementers MUST ensure that
+                properties necessary for the validation and verification of a credential are NOT selectively
+                disclosable. Such properties include but are not limited to <a data-cite="VC-DATA-MODEL-2.0#status">
+                <code>credentialStatus</code></a> and <a data-cite="VC-DATA-MODEL-2.0#data-schemas">
+                <code>credentialSchema</code></a>.
```
Comment (TallTed) [inline index.html:528]:
```suggestion
                When securing verifiable presentations with [[SD-JWT]] implementers MUST ensure that
                properties necessary for the validation and verification of a credential are NOT selectively
                disclosable (i.e., such properties MUST be disclosed). These properties include but are not
                limited to <a data-cite="VC-DATA-MODEL-2.0#status"> <code>credentialStatus</code></a>
                and <a data-cite="VC-DATA-MODEL-2.0#data-schemas"> <code>credentialSchema</code></a>.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Tweak for clarity.

**Review by selfissued — APPROVED** [review]
LGTM

**Review by OR13 — APPROVED** [review]

**Comment by decentralgabe** [comment]
Open for over a week. Editorial approval. Changes adopted. Merging.

**Pull request merged**